pyspark/dl/optim/optimizer.py

```diff
@@ -188,6 +188,14 @@ def setvalidation(self, batch_size, val_rdd, trigger, val_method=["Top1Accuracy"
         callBigDlFunc(self.bigdl_type, "setValidation", self.value, batch_size,
                       trigger, val_rdd, val_method)
 
+    def setmodel(self, model):
```

Comment:
From the issue, I think you should add another prepare_input method. And in python, the naming style should be set_model instead of setmodel